features/SideNavigation.feature

```diff
@@ -0,0 +1,165 @@
+            | Core Category Section                       | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | N/A              |
+            | Core Category Section with navigation arrow | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | to the right     |
+            | Parent                                      | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | N/A              |
+            | Parent with dropdown arrow                  | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | down             |
+            | Child                                       | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | 003B45 (dark green) | N/A              |
+            | Child with dropdown arrow                   | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | 003B45 (dark green) | down             |
+
+    @ui
+    @motivating
+    Scenario Outline: Hovered menu item state in the side navigation bar
+        Given a user hovers over the <menuItem> menu item in the side navigation bar
+        Then they see that the background color is <backgroundColor>
+        And text color is <textColor>
+        And text type is  <textType>
+        And the icon color is <iconColor>
+        And the dropdown line is <dropdownLine>
+
+        Examples:
+
+            | <menuItem>                                  | <backgroundColor>    | <textColor>         | <textType> | <iconColor>         | <dropdownLine>      | <arrowDirection> |
+            | Core Category Section                       | E7EDEE (muted green) | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | N/A              |
+            | Core Category Section with navigation arrow | E7EDEE (muted green) | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | to the right     |
+            | Parent                                      | E7EDEE (muted green) | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | N/A              |
+            | Parent with dropdown arrow                  | E7EDEE (muted green) | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | down             |
+            | Child                                       | E7EDEE (muted green) | 003B45 (dark green) | normal     | 003B45 (dark green) | 003B45 (dark green) | N/A              |
+            | Child with dropdown arrow                   | E7EDEE (muted green) | 003B45 (dark green) | normal     | 003B45 (dark green) | 003B45 (dark green) | down             |
+
+    @ui
+    @motivating
+    Scenario Outline: Selected menu item state in the side navigation bar
```

Comment:
Are all the states you have here testable given that when you select one of the core category sections, it redirects you?